SentinelOneV2 (#31687)
demisto/content: Packs/SentinelOne/ReleaseNotes/3_2_15.md

#### Integrations

##### SentinelOne v2

> - Bug fix on the ***sentinelone-run-remote-script*** command, removing the empty values from the payload of an API call.

Reviewer: ```suggestion
Fixed bug for the ***sentinelone-run-remote-script*** command, removing the empty values from the payload of an API call.
```
